incremental-font-transfer/Cargo.toml

```diff
@@ -43,3 +43,8 @@ path = "src/lib.rs"
 name = "ift_extend"
 path = "src/ift_extend.rs"
 required-features = ["cli"]
+
+[[bin]]
+name = "ift_graph"
+path = "src/ift_graph.rs"
```

Comment:
missed this the first time, but by convention bins live in src/bin, in which case you don't need to specify the path here: https://doc.rust-lang.org/1.82.0/cargo/reference/cargo-targets.html#binaries